PIL/Image.py

```diff
@@ -592,6 +592,12 @@ def _copy(self):
         self.pyaccess = None
         self.readonly = 0
 
+    def _mutable(self):
+        if self.readonly:
```

Comment:
I don't like the name of this, as it shows in the calling code. The bare call to `self._mutable` while clearly having a side effect, doesn't clearly explain what's going on. 

I'd be happier with something like `_ensure_mutable`.